2013/07/23 07:19:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "projects".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('projects')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('projects')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/projects')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/projects
HTTP_REFERER=http://localhost/demo/index.php?r=site/index
---
2013/07/23 07:19:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "profile".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('profile')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('profile')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/profile')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/profile
HTTP_REFERER=http://localhost/demo/index.php?r=site/projects
---
2013/07/23 07:19:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "publications".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('publications')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('publications')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/publicatio...')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/publications
HTTP_REFERER=http://localhost/demo/index.php?r=site/index
---
2013/07/23 07:19:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "team".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('team')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('team')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/team')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/team
HTTP_REFERER=http://localhost/demo/index.php?r=site/publications
---
2013/07/23 07:19:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "gallery".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('gallery')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('gallery')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/gallery')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/gallery
HTTP_REFERER=http://localhost/demo/index.php?r=site/team
---
2013/07/23 07:19:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "news".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('news')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('news')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/news')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/news
HTTP_REFERER=http://localhost/demo/index.php?r=site/gallery
---
2013/07/23 07:20:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "projects".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('projects')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('projects')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/projects')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/projects
HTTP_REFERER=http://localhost/demo/index.php?r=site/login
---
2013/07/23 07:24:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "projects".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('projects')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('projects')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/projects')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/projects
HTTP_REFERER=http://localhost/demo/index.php?r=site/login
---
2013/07/23 07:24:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "projects".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('projects')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('projects')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/projects')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/projects
HTTP_REFERER=http://localhost/demo/index.php?r=site/index
---
2013/07/23 07:24:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "profile".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('profile')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('profile')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/profile')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/profile
HTTP_REFERER=http://localhost/demo/index.php?r=site/projects
---
2013/07/23 07:24:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "surveys".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('surveys')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('surveys')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/surveys')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/surveys
HTTP_REFERER=http://localhost/demo/index.php?r=site/profile
---
2013/07/23 07:24:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "publications".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('publications')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('publications')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/publicatio...')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/publications
HTTP_REFERER=http://localhost/demo/index.php?r=site/surveys
---
2013/07/23 07:24:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "careers".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('careers')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('careers')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/careers')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/careers
HTTP_REFERER=http://localhost/demo/index.php?r=site/publications
---
2013/07/23 07:30:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "team".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('team')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('team')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/team')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/team
HTTP_REFERER=http://localhost/demo/index.php?r=site/index
---
2013/07/23 07:30:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "news".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('news')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('news')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/news')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/news
HTTP_REFERER=http://localhost/demo/index.php?r=site/team
---
2013/07/23 07:35:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "projects".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('projects')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('projects')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/projects')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/projects
HTTP_REFERER=http://localhost/demo/index.php?r=site/news
---
2013/07/23 07:35:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "projects".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('projects')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('projects')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/projects')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/projects
HTTP_REFERER=http://localhost/demo/index.php?r=site/news
---
2013/07/23 07:35:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "projects".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('projects')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('projects')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/projects')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/projects
HTTP_REFERER=http://localhost/demo/index.php?r=site/projects
---
2013/07/23 07:35:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "profile".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('profile')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('profile')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/profile')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/profile
HTTP_REFERER=http://localhost/demo/index.php?r=site/projects
---
2013/07/23 07:35:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "surveys".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('surveys')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('surveys')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/surveys')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/surveys
HTTP_REFERER=http://localhost/demo/index.php?r=site/profile
---
2013/07/23 07:35:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "publications".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('publications')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('publications')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/publicatio...')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/publications
HTTP_REFERER=http://localhost/demo/index.php?r=site/surveys
---
2013/07/23 07:35:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "careers".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('careers')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('careers')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/careers')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/careers
HTTP_REFERER=http://localhost/demo/index.php?r=site/publications
---
2013/07/23 07:35:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "team".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('team')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('team')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/team')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/team
HTTP_REFERER=http://localhost/demo/index.php?r=site/careers
---
2013/07/23 07:35:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "gallery".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('gallery')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('gallery')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/gallery')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/gallery
HTTP_REFERER=http://localhost/demo/index.php?r=site/team
---
2013/07/23 07:35:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "news".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('news')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('news')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/news')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/news
HTTP_REFERER=http://localhost/demo/index.php?r=site/gallery
---
2013/07/23 07:35:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "projects".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('projects')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('projects')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/projects')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/projects
HTTP_REFERER=http://localhost/demo/index.php?r=site/page&view=about
---
2013/07/23 07:36:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "projects".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('projects')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('projects')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/projects')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/projects
HTTP_REFERER=http://localhost/demo/index.php?r=site/projects
---
2013/07/23 07:36:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "profile".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('profile')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('profile')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/profile')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/profile
HTTP_REFERER=http://localhost/demo/index.php?r=site/projects
---
2013/07/23 07:36:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "surveys".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('surveys')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('surveys')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/surveys')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/surveys
HTTP_REFERER=http://localhost/demo/index.php?r=site/profile
---
2013/07/23 07:37:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "projects".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('projects')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('projects')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/projects')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/projects
HTTP_REFERER=http://localhost/demo/index.php?r=site/index
---
2013/07/23 07:38:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "projects".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('projects')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('projects')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/projects')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/projects
HTTP_REFERER=http://localhost/demo/index.php?r=site/index
---
2013/07/23 07:38:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "profile".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('profile')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('profile')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/profile')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/profile
HTTP_REFERER=http://localhost/demo/index.php?r=site/projects
---
2013/07/23 07:38:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "publications".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('publications')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('publications')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/publicatio...')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/publications
HTTP_REFERER=http://localhost/demo/index.php?r=site/profile
---
2013/07/23 07:38:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "careers".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('careers')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('careers')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/careers')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/careers
HTTP_REFERER=http://localhost/demo/index.php?r=site/publications
---
2013/07/23 07:38:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "team".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('team')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('team')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/team')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/team
HTTP_REFERER=http://localhost/demo/index.php?r=site/careers
---
2013/07/23 07:38:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "gallery".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('gallery')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('gallery')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/gallery')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/gallery
HTTP_REFERER=http://localhost/demo/index.php?r=site/team
---
2013/07/23 07:38:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "news".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('news')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('news')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/news')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/news
HTTP_REFERER=http://localhost/demo/index.php?r=site/gallery
---
2013/07/23 07:43:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "surveys" was not found.' in C:\xampp\htdocs\yii\yii\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('surveys')
#1 C:\xampp\htdocs\yii\yii\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\yii\yii\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\yii\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\yii\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#8 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/demo/index.php?r=site/page&view=surveys
HTTP_REFERER=http://localhost/demo/index.php?r=site/page&view=about
---
2013/07/23 07:43:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "surveys" was not found.' in C:\xampp\htdocs\yii\yii\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('surveys')
#1 C:\xampp\htdocs\yii\yii\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\yii\yii\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\yii\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\yii\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#8 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/demo/index.php?r=site/page&view=surveys
HTTP_REFERER=http://localhost/demo/index.php?r=site/page&view=surveys
---
2013/07/23 07:43:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "news" was not found.' in C:\xampp\htdocs\yii\yii\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('news')
#1 C:\xampp\htdocs\yii\yii\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\yii\yii\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\yii\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\yii\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#8 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/demo/index.php?r=site/page&view=news
HTTP_REFERER=http://localhost/demo/index.php?r=site/page&view=gallery
---
2013/07/23 08:10:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "news" was not found.' in C:\xampp\htdocs\yii\yii\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('news')
#1 C:\xampp\htdocs\yii\yii\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\yii\yii\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\yii\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\yii\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#8 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/demo/index.php?r=site/page&view=news
HTTP_REFERER=http://localhost/demo/index.php?r=site/page&view=gallery
---
2013/07/24 08:13:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "news" was not found.' in C:\xampp\htdocs\yii\yii\framework\web\actions\CViewAction.php:112
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\actions\CViewAction.php(123): CViewAction->resolveView('news')
#1 C:\xampp\htdocs\yii\yii\framework\web\actions\CAction.php(76): CViewAction->run()
#2 C:\xampp\htdocs\yii\yii\framework\web\CController.php(308): CAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\yii\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#4 C:\xampp\htdocs\yii\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('page')
#6 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#7 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#8 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/demo/index.php?r=site/page&view=news
HTTP_REFERER=http://localhost/demo/index.php?r=site/page&view=gallery
---
2013/07/24 08:15:28 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'testdrive'' in C:\xampp\htdocs\yii\yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\yii\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\yii\yii\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii\yii\framework\gii\generators\model\ModelCode.php(57): CModule->__get('db')
#5 C:\xampp\htdocs\yii\yii\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\yii\yii\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\yii\yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\yii\yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\yii\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('')
#13 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/demo/index.php?r=gii/model
HTTP_REFERER=http://localhost/demo/index.php?r=gii/model
---
2013/07/24 08:15:47 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'testdrive'' in C:\xampp\htdocs\yii\yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\yii\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\yii\yii\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii\yii\framework\gii\generators\model\ModelCode.php(57): CModule->__get('db')
#5 C:\xampp\htdocs\yii\yii\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\yii\yii\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\yii\yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\yii\yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\yii\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('')
#13 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/demo/index.php?r=gii/model
HTTP_REFERER=http://localhost/demo/index.php?r=gii
---
2013/07/24 08:19:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "gii".' in C:\xampp\htdocs\yii\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\CController.php(270): CController->missingAction('gii')
#1 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('gii')
#2 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/gii')
#3 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/demo/index.php?r=site/gii
---
2013/07/24 08:20:23 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'test.contact' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `contact`.
in C:\xampp\htdocs\demo\index.php (13)
2013/07/24 08:21:47 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'hrdp.contact' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `contact`.
in C:\xampp\htdocs\demo\index.php (13)
2013/07/24 08:21:50 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'hrdp.contact' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `contact`.
in C:\xampp\htdocs\demo\index.php (13)
2013/07/24 08:35:18 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'testdrive'' in C:\xampp\htdocs\yii\yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\yii\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\yii\yii\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii\yii\framework\gii\generators\model\ModelCode.php(57): CModule->__get('db')
#5 C:\xampp\htdocs\yii\yii\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\yii\yii\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\yii\yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\yii\yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\yii\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('')
#13 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/demo/index.php?r=gii/model
HTTP_REFERER=http://localhost/demo/index.php?r=gii/model
---
2013/07/24 08:35:45 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'testdrive'' in C:\xampp\htdocs\yii\yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\yii\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\yii\yii\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii\yii\framework\gii\generators\model\ModelCode.php(57): CModule->__get('db')
#5 C:\xampp\htdocs\yii\yii\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\yii\yii\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\yii\yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\yii\yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\yii\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('')
#13 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/demo/index.php?r=gii/model
HTTP_REFERER=http://localhost/demo/index.php?r=gii
---
2013/07/24 09:02:40 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('contactus/admin')
#9 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/demo/index.php?r=contactus/admin
HTTP_REFERER=http://localhost/demo/index.php?r=site/login
---
2013/07/24 09:04:34 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('contactus/admin')
#9 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/demo/index.php?r=contactus/admin
HTTP_REFERER=http://localhost/demo/index.php?r=contactus
---
2013/07/24 09:04:54 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('contactus/admin')
#9 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/demo/index.php?r=contactus/admin
HTTP_REFERER=http://localhost/demo/index.php?r=contactus/create
---
2013/07/24 09:06:53 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('contactus/admin')
#9 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/demo/index.php?r=contactus/admin
HTTP_REFERER=http://localhost/demo/index.php?r=contactus/create
---
2013/07/24 09:12:11 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('contactus/admin')
#9 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/demo/index.php?r=contactus/admin
HTTP_REFERER=http://localhost/demo/index.php?r=contactus/create
---
2013/07/24 09:46:57 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'hrdp.tbl_user' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `tbl_user`.
in C:\xampp\htdocs\demo\index.php (13)
2013/07/24 09:46:57 [error] [exception.CDbException] exception 'CDbException' with message 'The table "tbl_user" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\yii\framework\db\ar\CActiveRecord.php:2310
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\yii\framework\gii\generators\crud\CrudCode.php(66): CActiveRecord::model('User')
#2 C:\xampp\htdocs\yii\yii\framework\validators\CInlineValidator.php(42): CrudCode->validateModel('model', Array)
#3 C:\xampp\htdocs\yii\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(CrudCode), 'model')
#4 C:\xampp\htdocs\yii\yii\framework\base\CModel.php(159): CValidator->validate(Object(CrudCode), NULL)
#5 C:\xampp\htdocs\yii\yii\framework\gii\CCodeGenerator.php(159): CModel->validate()
#6 C:\xampp\htdocs\yii\yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#7 C:\xampp\htdocs\yii\yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#8 C:\xampp\htdocs\yii\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('')
#12 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/crud')
#13 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/demo/index.php?r=gii/crud
HTTP_REFERER=http://localhost/demo/index.php?r=gii/crud
---
2013/07/24 09:47:31 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'hrdp.tbl_user' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `tbl_user`.
in C:\xampp\htdocs\demo\index.php (13)
2013/07/24 09:47:31 [error] [exception.CDbException] exception 'CDbException' with message 'The table "tbl_user" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\yii\framework\db\ar\CActiveRecord.php:2310
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\yii\framework\gii\generators\crud\CrudCode.php(66): CActiveRecord::model('User')
#2 C:\xampp\htdocs\yii\yii\framework\validators\CInlineValidator.php(42): CrudCode->validateModel('model', Array)
#3 C:\xampp\htdocs\yii\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(CrudCode), 'model')
#4 C:\xampp\htdocs\yii\yii\framework\base\CModel.php(159): CValidator->validate(Object(CrudCode), NULL)
#5 C:\xampp\htdocs\yii\yii\framework\gii\CCodeGenerator.php(159): CModel->validate()
#6 C:\xampp\htdocs\yii\yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#7 C:\xampp\htdocs\yii\yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#8 C:\xampp\htdocs\yii\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('')
#12 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/crud')
#13 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/demo/index.php?r=gii/crud
HTTP_REFERER=http://localhost/demo/index.php?r=gii/crud
---
2013/07/24 09:50:19 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'hrdp.tbl_user' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `tbl_user`.
in C:\xampp\htdocs\demo\index.php (13)
2013/07/24 09:50:19 [error] [exception.CDbException] exception 'CDbException' with message 'The table "tbl_user" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\yii\yii\framework\db\ar\CActiveRecord.php:2310
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\yii\yii\framework\gii\generators\crud\CrudCode.php(66): CActiveRecord::model('User')
#2 C:\xampp\htdocs\yii\yii\framework\validators\CInlineValidator.php(42): CrudCode->validateModel('model', Array)
#3 C:\xampp\htdocs\yii\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute(Object(CrudCode), 'model')
#4 C:\xampp\htdocs\yii\yii\framework\base\CModel.php(159): CValidator->validate(Object(CrudCode), NULL)
#5 C:\xampp\htdocs\yii\yii\framework\gii\CCodeGenerator.php(159): CModel->validate()
#6 C:\xampp\htdocs\yii\yii\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#7 C:\xampp\htdocs\yii\yii\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#8 C:\xampp\htdocs\yii\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('')
#12 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/crud')
#13 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/demo/index.php?r=gii/crud
HTTP_REFERER=http://localhost/demo/index.php?r=gii/crud
---
2013/07/24 09:53:00 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\yii\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user/admin')
#9 C:\xampp\htdocs\yii\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\demo\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/demo/index.php?r=user/admin
---
